huey/storage.py

```diff
@@ -104,8 +104,17 @@ def flush_all(self):
 
 
 class RedisStorage(BaseStorage):
-    def __init__(self, name='huey', blocking=False, read_timeout=1,
-                 max_errors=1000, connection_pool=None, **connection_params):
+    def __init__(self, name='huey', blocking=False, read_timeout=1, max_errors=1000,
+            connection_pool=None, url=None, **connection_params):
+        if sum(bool(x) for x in (url, connection_pool, connection_params)) > 1:
```

Comment:
`len` is semantically what you are interested in. The use of `sum` is predicated on the strange fact that Python treats `True` as `1`.